commit 90cfbcc5c1dec4b110afc01f9edb272b57951e7c
Author: Henrik Ingo <henrik.ingo@avoinelama.fi>
Date:   Fri Mar 17 00:48:31 2017 +0000

    uvcvideo: uvc_scan_fallback() for webcams with broken chain
    
    [ Upstream commit e950267ab802c8558f1100eafd4087fd039ad634 ]
    
    Some devices have invalid baSourceID references, causing uvc_scan_chain()
    to fail, but if we just take the entities we can find and put them
    together in the most sensible chain we can think of, turns out they do
    work anyway. Note: This heuristic assumes there is a single chain.
    
    At the time of writing, devices known to have such a broken chain are
      - Acer Integrated Camera (5986:055a)
      - Realtek rtl157a7 (0bda:57a7)
    
    Signed-off-by: Henrik Ingo <henrik.ingo@avoinelama.fi>
    Signed-off-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
    Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1445_90cfbcc5c1dec4b110afc01f9edb272b57951e7c.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1445_90cfbcc5c1dec4b110afc01f9edb272b57951e7c.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
